src/component/handlers/edit/editOnInput.js

```diff
@@ -42,22 +43,56 @@ function editOnInput(editor: DraftEditor): void {
 
   // We have already updated our internal state appropriately for this input
   // event. See editOnBeforeInput() for more info
-  if (editor._waitingOnInput) {
-    if (!editor._renderNativeContent) {
-      return;
+  if (editor._updatedNativeInsertionBlock && !editor._renderNativeContent) {
+    editor._updatedNativeInsertionBlock = false;
```

Comment:
We should set this to `null` so that it is either an object or `null` rather than an object or `false`